### PR TITLE
[RFR] Added “Redux” entry

### PR DESCRIPTION
**Pull request by CrocoDillon**
Same with Flux, tried to keep it minimal and not too confusing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
 
 [React](https://facebook.github.io/react/) section to be completed.
 
+## Redux
+
+[Redux](http://redux.js.org/) is a predictable state container for JavaScript apps, which is a fancy way of saying it controls application state and state mutations. It does so by keeping state in a store, which is the single source of truth.
+
+Redux is an alternative to Flux and used a lot together with React, but you can use it with any other view library.
```
Comment (KittyGiraudel) [inline README.md]: Making _Flux_ a link to `#flux` would be nice. :)
Making _React_ a link to `#react` would be nice. :)

Comment (CrocoDillon) [inline README.md]: Done!

**Comment by KittyGiraudel** [comment]
Thanks!

**Comment by CrocoDillon** [comment]
:v:

**Pull request merged**